src/dc_receive_imf.rs

```diff
@@ -3140,10 +3185,94 @@ mod tests {
         chat_id.accept(&t).await.unwrap();
         let chat = Chat::load_from_db(&t.ctx, chat_id).await.unwrap();
         assert_eq!(chat.name, "delta-dev");
+        assert!(chat.can_send(&t).await.unwrap());
```

Comment:
fwiw i still find tests returning `Result` really hard to work with since you don't get errors pointing to the right line.  `.unwrap()` might be a bit more verbose but much nicer to debug.